src/rules/converters/eslint-plugin-react/jsx-no-bind.ts

```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../../converter";
+
+export const convertJsxNoBind: RuleConverter = () => {
+    return {
```

Comment:
Reading through https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md, it looks like the new rule also checks for arrow functions. I think you'll want to add a `notices` member to this object to mention that the rule will now also check for them, right?